### DISCUSSION TITLE
Current fluid streaming performance.

### DISCUSSION THREAD

**Discussion by dolceAlka** [open-webui/open-webui#6873] [Issues]
Using cerebras AI, the api returns big chunks of data really fast. Using timing between inlet and outlet calling, its around 500 T/s without fluid streaming. the current fluid streaming ts function splits it into 3,4 word chunks, which heavily limits actual T/s to around 80 or 90ish. I built the image with the chunk size at 10 and I can manage around 150 T/s in webui performance. It would be nice if chunk size was togglable or was larger by default to support faster inference.

**Comment by 0xGitGuy** [comment]
Do you have a working Open Web UI function for Cerebras that you can share?
